simulator/src/resim/cmd_publish.rs

```diff
@@ -68,7 +67,7 @@ impl Publish {
     ) -> Result<(), Error> {
         let mut ledger = RadixEngineDB::with_bootstrap(get_data_dir()?);
         let mut executor = TransactionExecutor::new(&mut ledger, self.trace);
-        let package = Package::new(code);
+        let package = extract_package(code).unwrap();
```

Comment:
Minor: can we return the error code? The CLI has a cohesive way of dealing with errors.